src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html

```diff
@@ -3,7 +3,7 @@
     [style.max-width]="'780px'"
     [class.alert-success]="feedback.credits! > 0"
     [class.alert-danger]="feedback.credits! < 0"
-    [class.alert-warning]="feedback.credits === 0"
+    [class.alert-warning]="feedback.credits === 0 || feedback.credits == null"
```

Comment:
You can just do this:
```suggestion
    [class.alert-warning]="!feedback.credits"
```

Reply:
Yeah, you are right! Thanks